install/linux/usr/share/odemis/hwtest/tmcm-can-ctrl.odm.yaml

```diff
@@ -0,0 +1,20 @@
+SimOptical: {
+    class: Microscope,
+    role: brightfield,
+    children: [Light, TMCM_CAN],
+}
+
+Light: {
+    class: simulated.Light,
+    role: brightlight,
+}
```

Comment:
Actually, you can drop this. It was there to be able to start the GUI, but as there is no stage and no CCD, that's a bit hopeless anyway.